distribution/lib/Standard/Table/0.0.0-dev/src/Table.enso

```diff
@@ -2472,9 +2472,9 @@ type Table
         unique.mark_used self.column_names
 
         resolved = case value of
+            _ : Column -> value
             _ : Text -> self.make_constant_column value
             _ : Expression -> self.evaluate_expression value on_problems
-            _ : Column -> value
```

Comment:
Again, intersection types have changed something fundamental about Enso semantics.

`case of` branches that used to be completely disjoint - a value was able to match only one of them - are now no longer disjoint - a multi-value can match multiple branches. So now we need to be more careful about ordering of the branches. If a value can match more than one, which branch is the one that should be preferred?

E.g. here single-text-value column would match both `_ : Text` and `_ : Column` branch (because it is `Column & Text`). Now, we want it to still go to the `_ : Column` branch, if it went to `_ : Text` branch that was leading to errors.

Reply:
- known since https://github.com/enso-org/enso/pull/11850#discussion_r1883374897
- [documented](https://github.com/enso-org/enso/blob/32bdcebec4f4a4ac40eb8ba97af73855cf4bdd7b/docs/types/intersection-types.md#narrowing-type-check) as _"Remember to use `f.sqrt` and not `c.sqrt`."_